### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dairer**

### CODE REVIEW

**Review by NiallJoeMaher — COMMENTED** [review]
Check the review comment @dairer

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # open-source-tutorial
+Am i doing this right? hmmm
```
Comment (NiallJoeMaher) [inline README.md]: Leave your username here 👍

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dairer) [reply README.md]: dairer

Comment (NiallJoeMaher) [reply README.md]: @dairer I meant on the file 😂

Comment (dairer) [reply README.md]: hahaha ok give me a sec

**Review by NiallJoeMaher — APPROVED** [review]
🌮

**Pull request merged**